### DISCUSSION TITLE
Windows Installer failing VirusTotal checks

### DISCUSSION THREAD

**Discussion by darth-kcaj** [LizardByte/community#307] [Sunshine] [answered]
Hey this is more of an FYI than anything else- the windows host installer for moonlight is failing a couple of virus total checks at time of writing. 

https://www.virustotal.com/gui/file/95d22f07256798c2c1f32e813e9ee2fc2888428ffc431b92d6c0ce07c80e2257

I know these things are very prone to throwing out the odd false positive here and there, but just flagging anyway...

**Comment by jakolion** [comment, marked as answer]
When uploading a few of the latest windows installers to virustotal.com, there are mostly 2-4 vendors reporting findings. Currently for the latest _sunshine-windows-installer.exe_ (v2024.1204.170534) there are two flags:

https://www.virustotal.com/gui/file/e38d44a6ab4da73cfe2ac5809b7a0826c84a048efd0d012b5d50138ccd11b309
**Trapmine:** Suspicious.low.ml.score
**Zillya:** Trojan.Convagent.Win32.574852

The release before was flagged additionally by **Palo Alto Networks:** Generic.ml

The _sunshine-windows-portable.zip_ is fine: https://www.virustotal.com/gui/file/61e2333978c63e4b57a2d959af495f31ad87d0cdcc967db4b6064194e6880820

So maybe it's something with the installer?
Are there any detailed explanations why these scanners are flagging it? I'm sure these are all false positives.

[a comment by an automated account is not shown]
